free-3.md

```diff
@@ -57,12 +120,35 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This project seems to have the most uncertainties of the proposed projects.
+This is mostly because a lot of this project involves streamlining a number of
+APIs, codecs, and tools under a single roof. Because of this, unless extensive
+use of existing code from something like Foobar2000 can be utilized, a lot of
+this project, although not all of it, will be on aspects besides the
+language design. Although the language is ultimately the glue that binds
+everything together, and thus its design will be closely tied to how the
+various APIs work, the more I'm thinking about it, the more this might
+devolve into trying to make a bunch of APIs play nicely together. That being
+said, I would need to investigate code libraries a bit more before I can
+be certain of this.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Like  ImageOrganizer, although all the GUIs might make this, overall, a very
+ambitious project, by scaling to specific functionality, assuming the existing
+APIs play nice, feels like this could be a very reasonable project. This is
+especially so since the ambitious parts of the project (i.e. the APIs) can be
+trimmed down if they are not cooperating without limiting the design of the
+language.
```

Comment:
Like I said before, of the projects you proposed, this one seems to have the least actual language work. Maybe that's actually a good thing, as it would scope well to a DSL project.